markdown.html.markdown

```diff
@@ -153,12 +153,13 @@ inside your code -->
 John didn't even know what the `go_to()` function did!
 
 <!-- In Github Flavored Markdown, you can use a special syntax for code -->
-
-\`\`\`ruby <!-- except remove those backslashes when you do this, just ```ruby ! -->
-def foobar
-    puts "Hello world!"
-end
-\`\`\` <!-- here too, no backslashes, just ``` -->
+(the extra indentation is only to render the fenced block as-is here.
+omit it in your markdown) -->
```

Comment:
Is the `-->` supposed to be here? IMO, remove the statement from parentheses, as it is important and yet another markdown tip for the cheat sheet.